run.py

```diff
@@ -35,8 +36,8 @@ def runSolitude(solitudeProcess):
 
 def main():
     setEnviroment()
-    time.sleep(20)
-    solitudeWebProcess = subprocess.Popen([os.getenv('PYTHON_PATH'), os.getenv('WEBAPP_SCRIPT')], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # $, env=env)
+    #time.sleep(20)
```

Comment:
The sleep is needed to ensure that the database is fully initialized by the time the web app tries to connect to it.